modules/vstudio/vs2005_dotnetbase.lua

```diff
@@ -328,29 +328,27 @@
 -- Write out the references item group.
 --
 
-	dotnetbase.elements.references = function(prj)
+	dotnetbase.elements.references = function(prj, cfg)
 		return {
 			dotnetbase.assemblyReferences,
 			dotnetbase.nuGetReferences,
```

Comment:
Shouldn't we add `cfg` parameter to `dotnetbase.nuGetReferences` even if unused.